src/openrct2/world/Scenery.cpp

```diff
@@ -197,6 +199,8 @@ void scenery_remove_ghost_tool_placement()
     {
         gSceneryGhostType &= ~SCENERY_GHOST_FLAG_1;
         TileElement* tileElement = map_get_first_element_at(x / 32, y / 32);
+        if (tileElement == nullptr)
+            return;
```

Comment:
we will want the rest of this function to be called. I know the rest will also fail but it needs to clear the ghost flags and such and that will be the most clean code. So move this into the do while and make it a break.